.eslintrc

```diff
@@ -0,0 +1,57 @@
+{
+  "root": true,
+  "extends": ["eslint:recommended", "plugin:react/recommended", "airbnb-base"],
+  "env": {
+    "node": true,
+    "es6": true,
+    "browser": true,
+    "jest": true
+  },
+  "globals": {
+    "expect": false
```

Comment:
why are you doing this? You already have jest in your env there is no need for this